notifiers/slack.py

```diff
@@ -11,6 +13,9 @@ def __init__(self, config):
 
         self._webhook_url = config['webhook_url']
 
+        if self._webhook_url == 'env' or environ.get('SLACK_WEBHOOK_URL') != '':
```

Comment:
Since we support both Slack and Teams, could we rename this to `UPSTREAM_WEBHOOK_URL` or something similar?

Reply:
i think this was just because it was being used in the slack.py.